server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -711,6 +901,88 @@ boolean blackhole() {
                 assert blackholed || unDisconnected == false;
                 return blackholed;
             }
+
+            ClusterState getLastAppliedClusterState() {
```

Comment:
This is unused.

Reply:
I've now made use of it by turning most assertions on the applier state to actually use the applied state (e.g. after stabilization).